AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_AutoComponent_RPC.py

```diff
@@ -73,7 +73,19 @@ def Multiplayer_AutoComponent_RPC():
         helper.succeed_if_log_line_found('EditorServer', "Script: AutoComponent_RPC_NetLevelEntity: Authority sending RPC to play some fx.", section_tracer.prints, PLAYFX_RPC_WAIT_TIME_SECONDS)
         helper.succeed_if_log_line_found('Script', "AutoComponent_RPC_NetLevelEntity: I'm a client playing some fx.", section_tracer.prints, PLAYFX_RPC_WAIT_TIME_SECONDS)
 
-    
+        # Autonomous->Authority RPC
```

Comment:
nit: this looks like we're sending two RPCs? AutonomousToAuthorityNoParam and AutonomousToAuthority with a parm? Comments are a little unclear.

Also "AutonomousToAuthority RPC" in send but "AutonomousToAuthorityRPC" in recieve, looks like inconsistent log output, as everything else has " RPC"

Reply:
Good eye! Fixed!